pkg/iac/terraform/state/enumerator/state_enumerator.go

```diff
@@ -10,13 +13,49 @@ type StateEnumerator interface {
 	Enumerate() ([]string, error)
 }
 
+type EnvProxy struct {
```

Comment:
I think this should be put in another package as there is nothing related to IaC stuff. WDYT of `pkg/helpers` @sundowndev @lotoussa ? (even if I don't really like "helpers" naming since it can quickly become a hold-all ...)

Reply:
This is convenient

Reply:
@eliecharra I prefer having a dedicated pkg for piece of code like this one, I think it fits more with the Go design patterns. But helpers can also be great, as long as it's not about extending every other packages (which can quickly become messy).

So I suggest we put this in `pkg/envproxy` or something like `pkg/iac/terraform/state/helpers`